extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -125,6 +126,38 @@ export const LIFECYCLE_STAGES = {
 };
 
 
+/**
+ * Utility function that ensures any error thrown is handled by optional
+ * onError handler (if none provided, error is swallowed).
+ * @param {!Function} fn to protect
+ * @param {T=} opt_this An optional object to use as the 'this' object
+*     when calling the function.
+ * @param {function(this:T, !Error, ...*):?=} opt_onError function given error
+ *    and arguments provided to function call.
+ * @return {!Function} protected function
+ * @template T
+ * @visibileForTesting
+ * @private
+ */
+export const protectFunctionWrapper_ = (fn, opt_this, opt_onError) => {
```

Comment:
Would probably make more sense for this to be `export function protectFunctionWrapper_(fn, opt_this, opt_onError) {`.

Also, old-style optional parameters [are no longer used](https://google.github.io/styleguide/jsguide.html#features-functions-parameters); rename them and give explicit defaults (which can be `undefined` if you want the same semantics) instead.